pkg/chunk/table_manager.go

```diff
@@ -126,6 +127,11 @@ type TableManager struct {
 // NewTableManager makes a new TableManager
 func NewTableManager(cfg TableManagerConfig, schemaCfg SchemaConfig, maxChunkAge time.Duration, tableClient TableClient,
 	objectClient BucketClient) (*TableManager, error) {
+	// Assume the newest config is the one to use for validation of retention
+	if cfg.RetentionPeriod != 0 && cfg.RetentionPeriod%schemaCfg.Configs[len(schemaCfg.Configs)-1].IndexTables.Period != 0 {
```

Comment:
This checks they are not zero.  Should it test to ensure they are multiples, like the error says?

Reply:
Sorry, I think you missed seeing that `%` operator between all the text or maybe I am not able to see the problem that you are seeing?

Reply:
Yeah you are right.  Sorry.